visallo/visallo
# when deleting a case, clean up workspace to entity edges and all sandboxed vertices

Testing Instructions:
Create a case with two graph work products
Add a person vertex to the first work product, note the vertex id
Delete the first work product -> diff panel should still say 1
Delete the case, make sure database marks the person vertex as soft deleted

Points of Regression: Deleting cases

CHANGELOG
Fixed: Deleting cases not cleaning up workspace to entity edges and sandboxed vertices
